### ISSUE TITLE
Data Manager: Documented Application API

### ISSUE THREAD

**Issue by alxhub** [angular/tactical#12]
Declare a documented API that will be exposed to the application for making requests to Tactical and receiving one or more responses.

**Comment by ttowncompiled** [comment]
### Tactical API - usable by the _application_
```typescript
get(key: Object): Object | null;
```
Returns an object associated with the provided key or **null** &mdash; if no object is associated with the provided key.

```typescript
observe(key: Object): Observable | null;
```
Returns an obserable associated with the provided key or **null** &mdash; if no object is associated with the provided key.

**Comment by alxhub** [comment]
Implemented in #24
